### PR TITLE
Java cc26

### PR DESCRIPTION
**Pull request by wafman**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401Challenges/src/main/java/code401Challenges/InsertionSort.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package code401Challenges;
+
+public class InsertionSort {
+
+    public static void main(String args[]){
```
Comment (evanslaton) [inline code401Challenges/src/main/java/code401Challenges/InsertionSort.java]: The main isn't needed!

File: code401Challenges/src/main/java/code401Challenges/InsertionSort.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    }
+
+    public static void InsertionSort(int [] arr){
+        if(arr.length == 0){
+            throw new IllegalArgumentException("cannot sort empty array");
+        }
```
Comment (evanslaton) [inline code401Challenges/src/main/java/code401Challenges/InsertionSort.java]: You could have added an extra conditional to check if the length of the array is 1 (no necessary)
